MathLib/LinAlg/Dense/DenseVector.h

```diff
@@ -75,6 +75,14 @@ class DenseVector : public std::valarray<T>
 		}
 	}
 
+	/// get entry values to an array
+	void getValues(T u[])
```

Comment:
I see that the current interface is probably due to the interface of `PropertyVector`. So maybe that can be changed later on.